APIs_Platform/FileHandle/main.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2006-2020 Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include "mbed.h"
+
+static DigitalOut led2(LED2);
+
+// BufferedSerial derives from FileHandle
+static BufferedSerial device(STDIO_UART_TX, STDIO_UART_RX);
+
+int main()
+{
+    // Perform device-specific setup
+    device.set_baud(19200);

```

Comment:
Changing the baud rate is not useful for this example.

Reply:
Removed and updated `README.md`.